opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.hpp

```diff
@@ -46,8 +46,11 @@ namespace Opm {
             const_iterator begin() const;
             const_iterator end() const;
 
+            static std::vector <std::string> getAllExpandedKeywords();
```

Comment:
Does it have to be? It's a constant list and it's duplication I'm fine with personally.